dispatcher.go

```diff
@@ -335,7 +335,7 @@ func (d *Dispatcher) Start() error {
 			errs = append(errs, newErrors...)
 		}
 
-		return errors.ErrorGroup(errs)
+		return multierr.Combine(errs...)
```

Comment:
We're not really exposing anything different to the users. It's an opaque error object with no other information on it. The contract for errors is always specified in the documentation because the types don't/can't enforce anything. So the contract here is the same as before: You get an error object.